test/functionalTest/cases/0117_convop_using_standard_ops/putSubscriptionConvOp.test

```diff
@@ -228,6 +228,9 @@ echo
 echo
 
 
+# Avoid throttling (1 second) for the subscription
+sleep 2
```

Comment:
Without this sleep, the coming update (11. Update E1/T1/A1) is ignored due to throttling.
This is how it MUST work, but it didn't before this PR.
So, this is actually a bug-fix in throttling.
I forgot about this, perhaps we should add this information in the CNR.